src/Service/MailService.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace App\Service;
+
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+class MailService
+{
+    private MailerInterface $mailer;
+
+    public function __construct(MailerInterface $mailer)
+    {
+        $this->mailer = $mailer;
+    }
+
+    public function sendExpiringRegistrationEmail(string $email, array $cars): void
+    {
+        // Inside HTML email
+        $emailBody = "
```

Comment:
instead of putting it here, use twig https://symfony.com/doc/current/mailer.html#twig-html-css